### PR TITLE
Refactoring for code convention

### PR DESCRIPTION
**Pull request by ancy0**
## Describe changes

Refactor the entire code according to the rules defined by the meeting

## Types of changes

What is the type of code change?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (changes that resolve errors)
- [ ] New feature (changes which adds functionality)
- [ ] Breaking change (big changes that affect existing functionality)
- [ ] Documentation Update

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the "README.md"
- [x] My code follows the style guidelines of this project
- [x] My changes generate no new warnings
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation
- [x] Any dependent changes have been merged and published in downstream modules

## Further comments

_Please let me know if there's anything else to explain_

### CODE REVIEW

File: src/components/common/InputComponent.js [hunk 1]
```diff
@@ -130,28 +126,27 @@ const InputFormControl = styled(FormControl)`
...
   align-items: center;
+  position: static;
   transform-origin: center left;
+  height: 100%;
   gap: 0.5rem;
-
+  color: ${COLOR.MAIN_BLACK};
   font-size: 1.8rem;
   font-weight: 500;
   text-align: center;
```
Comment (sinji2102) [inline src/components/common/InputComponent.js:143]: 👍🏻

**Review by sinji2102 — APPROVED** [review]
Good Refactoring 😄

File: src/components/step1/RequiredFieldContainer.js [hunk 2]
```diff
@@ -1,11 +1,10 @@
...
+import { useState, useEffect } from "react";
+import { useRecoilState } from "recoil";
 import { Grid } from "@mui/material";
+import axios from "axios";
+import { TextInputContainer } from "../common/InputComponent";
 import { SelectAuto } from "../common/SelectAuto";
-import { useRecoilState } from "recoil";
 import { repoDataAtomFamily } from "../../recoil/repoData";
 
 /* for Owner, RepoName, Description */
```
Comment (hummingbbird) [inline src/components/step1/RequiredFieldContainer.js:10]: a single annotation use "//"(not /* */) !!

File: src/components/step1/SelectContainer.js [hunk 3]
```diff
@@ -1,11 +1,10 @@
...
-
 import { useEffect, useState } from "react";
+import { useRecoilState } from "recoil";
+import { Grid, Typography } from "@mui/material";
 import axios from "axios";
 import { AutocompleteInput } from "../common/Autocomplete";
 import { repoDataAtomFamily } from "../../recoil/repoData";
-import { Grid, Typography } from "@mui/material";
 
 /* for select Language & Framework  */
```
Comment (hummingbbird) [inline src/components/step1/SelectContainer.js:9]: a single annotation use "//"(not /* */) !!

**Pull request merged**